whois-api/src/test/java/net/ripe/db/whois/api/rdap/AbstractRdapIntegrationTest.java

```diff
@@ -17,24 +17,20 @@
 import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.is;
 
-public abstract class AbstractRdapIntegrationTest extends AbstractIntegrationTest {
+public abstract class AbstractRdapIntegrationTest extends AbstractElasticSearchIntegrationTest {
 
```

Comment:
Please revert these changes, ES has a seperate profile and we need docker to run them

Reply:
Rdap uses ES.

Reply:
done